### ISSUE TITLE
TypeError on All Plots

### ISSUE THREAD

**Issue by WillKoehrsen** [plotly/plotly_express#15]
When trying to create any plot, I receive the error: `TypeError: super(type, obj): obj must be an instance or subtype of type`. This occurs even when running the examples as below:

```python
import plotly_express as px
gapminder = px.data.gapminder()
px.choropleth(gapminder, locations="iso_alpha", color="lifeExp", hover_name="country", animation_frame="year",
              color_continuous_scale=px.colors.sequential.Plasma, projection="natural earth")
```
<img width="915" alt="Screen Shot 2019-03-21 at 09 13 08 EDT" src="https://user-images.githubusercontent.com/17484616/54754367-b806d800-4bb9-11e9-92dd-9320effbb031.png">

Using `plotly-express==0.1`

**Comment by nicolaskruchten** [comment]
Thanks for the bug report! can you tell me which versions of Python, Jupyter and Plotly.py you're on?

**Comment by WillKoehrsen** [comment]
Python 3.6.5, Jupyter 1.0.0, and Plotly 3.7.1.

**Comment by WillKoehrsen** [comment]
Okay, the error is occurring with _any_ plot, not just the choropleth so I've updated the issue.

**Comment by nicolaskruchten** [comment]
This error is coming from Plotly.py rather than Plotly Express, which is weird... in the same environment, are you able to make any Plotly plots at all? with cufflinks or otherwise?

**Comment by WillKoehrsen** [comment]
Yes, regular `plotly` and `cufflinks` both are able to produce plots in the notebook.

**Comment by WillKoehrsen** [comment]
Okay, this is strange, but after restarting the notebook server, the plots are working fine.

**Comment by nicolaskruchten** [comment]
Hmmm that is very strange. Let's close it for now. I did make a mistake in packaging up the PyPI build which I've just corrected (and pushed up as 0.1.1) so maybe that will prevent others from seeing it.
